impl/src/main/java/org/jboss/forge/furnace/container/guice/modules/GuiceContainerModule.java

```diff
@@ -84,6 +77,7 @@ private void bindServices()
                   String line = sc.nextLine();
                   Class serviceType = classLoader.loadClass(line);
                   bind(serviceType).toConstructor(serviceType.getConstructor());
+                  expose(serviceType);
```

Comment:
PrivateModule will expose only classes explicitly marked as exposed - great feature for our usecase